test/Array/rlexe.xml

```diff
@@ -9,6 +9,7 @@
   <test>
     <default>
       <files>array_qsortr.js</files>
+      <tags>exclude_jshost</tags>
```

Comment:
I don't understand why is this an xplat-only test ?

Reply:
From the top of the test file:

```js
// Test our custom qsort_r implementation on xplat.
if (WScript.Platform && WScript.Platform.OS != "win32") {
```

Reply:
The problem with this and other tests in jshost is that WScript.Platform used to be not defined, which was the original way this passed -- I defined it to make the ICU tests more consistent, and in doing so made a bunch of these if (platform && platform.x) checks come into play for jshost.

Reply:
Nevermind, I had to check the test.
Why not use a `exclude_win32` tag instead of dynamically check for the platform ?
Also, any reason this test won't run on Windows ?

Why does the arraysize must be different on debug and release build ?
Is the size testing different part of the sort algorithm ?

Reply:
I would ask @obastemur for the original rationale -- there are enough of these cases where we should probably do a once-over on all of them and normalize them to tags and add whatever jshost needs for things that really should run on Windows

Reply:
I see, I guess this is fine for now to unblock the build at the very least